mailpoet/assets/js/src/subscribers/importExport/import/step_data_manipulation/assign_tags.tsx

```diff
@@ -0,0 +1,40 @@
+import { MailPoet } from 'mailpoet';
+import { useCallback } from 'react';
+import { TokenField } from '../../../../common/form/tokenField/tokenField';
+
+interface Props {
+  selectedTags: string[];
+  setSelectedTags: (value) => void;
+}
+
+export function AssignTags({
+  selectedTags,
+  setSelectedTags,
+}: Props): JSX.Element {
+  const handleChange = useCallback(
+    ({ value }): void => {
+      setSelectedTags(value);
+    },
+    [setSelectedTags],
+  );
+
+  const tags = window.mailpoet_tags.map((tag) => tag.name);
```

Comment:
We wanted to prevent adding more usages of `window` within the new code. I guess we could assign the tags on the MailPoet module as we do with many other items we pass from the server.

Reply:
I addressed this in this commit [10059c1](https://github.com/mailpoet/mailpoet/pull/4347/commits/10059c165b5dc9b8b0da6cea5405fda86c3ee17e)